controllers/manager/manager.go

```diff
@@ -444,6 +464,38 @@ func (m *PlatformManager) GetPlatformClient(namespace string) *gophercloud.Servi
 	return nil
 }
 
+func (m *PlatformManager) SendHostStrategyUpdate(host_strategy HostStrategyInfo) error {
+	m.HostStrategyUpdateChannel <- host_strategy
+	return nil
+}
+
+func (m *PlatformManager) ReceiveHostStrategyUpdate() HostStrategyInfo {
+	host_strategy := <-m.HostStrategyUpdateChannel
+	return host_strategy
+}
+
+func (m *PlatformManager) SendHostReconciliationTrigger(host_strategy HostStrategyInfo) error {
+	m.HostReconciliationChannel <- host_strategy
+	return nil
+}
+
+func (m *PlatformManager) ReceiveHostReconciliationTrigger() HostStrategyInfo {
+	host_strategy := <-m.HostReconciliationChannel
```

Comment:
Please avoid to use "channel" in the name, that is obviously a channel, no need to mention, the naming convention should be verb+object

Reply:
> Please avoid to use "channel" in the name, that is obviously a channel, no need to mention, the naming convention should be verb+object

Done.